src/app/content/practiceQuestions/components/ShowPracticeQuestions.tsx

```diff
@@ -82,34 +84,42 @@ export const StyledContentLink = styled.a`
 
 // tslint:disable-next-line: variable-name
 const ShowPracticeQuestions = () => {
-  const book = useSelector(contentSelectors.book);
+  const {book, page} = useSelector(contentSelectors.bookAndPage);
   const section = useSelector(pqSelectors.selectedSection);
   const questionsCount = useSelector(pqSelectors.questionsCount);
   const currentQuestionIndex = useSelector(pqSelectors.currentQuestionIndex);
   const selectedSectionHasPracticeQuestions = useSelector(pqSelectors.selectedSectionHasPracticeQuestions);
+  const locationFilters = useSelector(pqSelectors.practiceQuestionsLocationFilters);
   const linkToTheSection = React.useMemo(() => {
     return book && section ? getBookPageUrlAndParams(book, section).url : null;
   }, [book, section]);
+  const nextSection = React.useMemo(() => {
+    const currentSectionId = section ? section.id : page ? page.id : null;
+    return currentSectionId ? getNextPageWithPracticeQuestions(currentSectionId, locationFilters, book) : undefined;
+  }, [book, page, section, locationFilters]);
 
   return (
     <ShowPracticeQuestionsBody
       data-testid='show-practice-questions-body'
       data-analytics-region='PQ popup'
     >
       {section ? <SectionTitle dangerouslySetInnerHTML={{ __html: section.title }} /> : null}
-      <QuestionsWrapper>
-        <QuestionsHeader>
-          <FormattedMessage id='i18n:practice-questions:popup:questions'>
-            {(msg: string) => msg}
-          </FormattedMessage>
-        </QuestionsHeader>
-        <ProgressBar total={questionsCount} activeIndex={currentQuestionIndex} />
-        {
-          selectedSectionHasPracticeQuestions && currentQuestionIndex === null
-            ? <IntroScreen />
-            : null
-        }
-      </QuestionsWrapper>
+      {questionsCount === 0 && nextSection
```

Comment:
Indentation looks weird here

Reply:
hmm, how would you change it?

Reply:
oh I didn't mean this specific line, sorry, lines 110 and below